### DISCUSSION TITLE
mempool bitcoin only edition

### DISCUSSION THREAD

**Discussion by dkatwo** [mempool/mempool#2559] [Ideas]
Alternative version of mempool with all references to fiat removed. So for example the conversion of the fees in Fiiat

**Comment by softsimon** [comment]
You are free to open a PR

Some hints on how:
* Override the default PRICE_DATA_SERVER data with empty strings
* Don't send price data in the websocket if there are no price data in the backend
* Don't display the fiat price component if there is no price data
